sample/NCronJobSample/Program.cs

```diff
@@ -30,9 +41,9 @@
 
 app.UseHttpsRedirection();
 
-app.MapPost("/trigger-instant", (IInstantJobRegistry instantJobRegistry) =>
+app.MapPost("/trigger-instant", async (IInstantJobRegistry instantJobRegistry) =>
     {
-        instantJobRegistry.RunInstantJob<PrintHelloWorldJob>("Hello from instant job!");
+        await instantJobRegistry.RunInstantJob<TestCancellationJob>("Hello from instant job!");
```

Comment:
I am somewhat in favor of this (rather than the user just ignores pro-actively the `Task` object) as it would result i non-breaking changes.

Reply:
Actually it is a fire and forget, just that this is handled inside the CronRegistry.RunInstantJob<TJob>() method. I moved it there because it didn't make sense to me to wait for the Jobs' results as the nature of Jobs needs to be real asynchronous. If we need to get the results of the Job there are other ways to handle this like with callbacks or your notification approach but we'd have to make it a more durable, maybe use an in-memory Job Run repository.

Reply:
Fair - my suggestion is then not to return `Task.Completed` and make the function `void` again.
There is no difference between awaiting this and having it void

Reply:
> Fair - my suggestion is then not to return `Task.Completed` and make the function `void` again.
> There is no difference between awaiting this and having it void

You're right, there's no difference to the end user hitting that endpoint, I just thought it was a more idiomatic way of expressing the method. I think historically the best practice was to use void for fire-and-forget, but lately I've seen more discussions about returning Task. I'll change it back to void.

Reply:
I am/would be all on board if there is a difference between awaiting and not awaiting the task.